query/src/main/java/org/infinispan/query/impl/massindex/DistributedExecutorMassIndexer.java

```diff
@@ -45,7 +45,7 @@
  * @since 7.1
  */
 @MBean(objectName = "MassIndexer",
-       description = "Component that rebuilds the index from the cached data")
+       description = "Component that rebuilds the Lucene index from the cached data")
```

Comment:
Seems cleaner to not specify Lucene to me.

Reply:
It's actually just for Lucene, so why not. First reference to MassIndexer in user guide says: "Occasionally you might need to rebuild the Lucene index by reconstructing it from the data stored in the Cache. " :)   We'll surely wipe this whole package out in the future if we decide  to reshape Infinispan query into a pluggable indexing/querying strategy. So far Lucene rears its head in all places.